python/tests/fixtures/feature_monitoring_result_fixtures.json

```diff
@@ -2,28 +2,128 @@
   "get_via_feature_group": {
     "response": {
       "id": 42,
-      "feature_store_id": 67,
-      "config_id": 32,
-      "detection_stats_id": 333,
-      "reference_stats_id": 222,
-      "execution_id": 123,
-      "shift_detected": true,
+      "featureStoreId": 67,
```

Comment:
We should also have result fixture with only detection window to test the scheduled statistics code path